src/browserslist

```diff
@@ -0,0 +1,9 @@
+# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
```

Comment:
whats this?

Reply:
Autoprefixer is a tool that handles css prefixes for features that require the -webkit or -moz. When this is configured it will correctly add those prefixes to the browsers we have defined as our min

Reply:
cool!